userdocs/docs/grpc/types/proto.md

```diff
@@ -0,0 +1,388 @@
+# Protocol Documentation
```

Comment:
it would be good to have these tables autogenerated from code comments in the future, just so we don't have to remember to update them

Reply:
They are generated from the proto files.

Reply:
ah sweet